xcell/tests/test_mapper_planck_all.py

```diff
@@ -68,26 +69,11 @@ def test_get_signal_map():
                           xc.mappers.MapperP15CIB(get_config('P15CIB')),
                           xc.mappers.MapperLenzCIB(get_config('LenzCIB'))])
 def test_get_nl_coupled(m):
+    print(m.beam_info)
     nl = m.get_nl_coupled()
     assert np.mean(nl) < 0.001
 
 
-def test_get_beam():
-    # No beam
-    m = xc.mappers.MapperPlanckBase(get_config('base'))
-    beam = m.get_beam()
-    assert np.all(beam == 1.0)
-
-    # 15-arcmin beam
-    fwhm = 15.
-    m = xc.mappers.MapperPlanckBase(get_config('base'))
-    m.beam_info = fwhm
-    beam = m.get_beam()
-    ls = np.arange(3*m.nside)
-    bls = np.exp(-0.5*ls*(ls+1)*(fwhm*np.pi/180/60/2.355)**2)
-    assert np.allclose(beam, bls, atol=0, rtol=1E-3)
-
-
```

Comment:
Could we keep this test? It still makes sense to have it, right?

Reply:
It still is in the tests. It just has been moved to test_utils

Reply:
It still is in there. I moved it to utils.